packages/kit/src/runtime/client/client.js

```diff
@@ -1149,7 +1149,21 @@ export function create_client({ target, base, trailing_slash }) {
 			}
 		},
 
-		goto: (href, opts = {}) => goto(href, opts, []),
+		goto: (href, opts = {}) => {
+			if (opts.hasOwnProperty('keepfocus')) {
+				throw new Error(
+					'`keepfocus` has been renamed to `keepFocus` (note the difference in casing)'
+				);
+			}
+
+			if (opts.hasOwnProperty('noscroll')) {
```

Comment:
`property in opts` is equivalent for all intents and purposes but is more byte-efficient

```suggestion
			// TODO remove for 1.0
			if ('keepfocus' in opts) {
				throw new Error(
					'`keepfocus` has been renamed to `keepFocus` (note the difference in casing)'
				);
			}

			if ('noscroll' in opts) {
```